2019-12-14 15:42:21,702 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,702 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,702 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,702 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,705 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,712 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,718 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:42:21,725 [32mINFO[0m: Parrots 0.4.0 | Git hash: 4ec10596 | Compute hash: 3ec76b2d[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 2 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 1 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 4 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 7 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 5 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 0 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 6 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,461 [32mINFO[0m: => rank 3 of 8 jobs, in SH-IDC1-10-5-31-36[0m
2019-12-14 15:43:03,698 [32mINFO[0m: config file: 
{
  "seed": 99,
  "net": {
    "type": "resnet50",
    "kwargs": {
      "num_classes": 1000
    }
  },
  "dataset": {
    "train": {
      "meta_file": "/mnt/lustre/share/images/meta/train.txt",
      "image_dir": "/mnt/lustre/share/images/train",
      "reader": "pillow",
      "resize": null,
      "random_crop": null,
      "random_resize_crop": 224,
      "colorjitter": [
        0.2,
        0.2,
        0.2,
        0.1
      ],
      "mean": [
        0.485,
        0.456,
        0.406
      ],
      "std": [
        0.229,
        0.224,
        0.225
      ],
      "mirror": true
    },
    "test": {
      "meta_file": "/mnt/lustre/share/images/meta/val.txt",
      "image_dir": "/mnt/lustre/share/images/val",
      "reader": "pillow",
      "resize": 256,
      "center_crop": [
        224,
        224
      ],
      "colorjitter": null,
      "mean": [
        0.485,
        0.456,
        0.406
      ],
      "std": [
        0.229,
        0.224,
        0.225
      ],
      "mirror": false
    },
    "batch_size": 32,
    "workers": 4
  },
  "trainer": {
    "max_epoch": 120,
    "test_freq": 1,
    "log_freq": 100,
    "label_smooth": null,
    "optimizer": {
      "type": "SGD",
      "kwargs": {
        "lr": 0.1,
        "momentum": 0.9,
        "weight_decay": 0.0001
      }
    },
    "lr_scheduler": {
      "warmup_epochs": 0,
      "type": "MultiStepLR",
      "kwargs": {
        "milestones": [
          30,
          60,
          90
        ],
        "gamma": 0.1
      }
    }
  },
  "saver": {
    "pretrain_model": null,
    "resume_model": null,
    "save_dir": "checkpoints",
    "save_epoch_freq": 1,
    "save_latest": false,
    "save_best": false,
    "sync_bn_stats": false
  },
  "monitor": {
    "type": "pavi",
    "_taskid": null,
    "kwargs": {
      "project": "default",
      "task": "resnet50",
      "model": "resnet50"
    }
  }
}[0m
2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m


2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m2019-12-14 15:43:03,657 [31mERROR[0m: Uncaught exception: 
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'[0m



Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
Traceback (most recent call last):
  File "../tools/main.py", line 291, in <module>
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    main()
  File "../tools/main.py", line 67, in main
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
    logger().info("config file: \n{}".format(
AttributeError: 'NoneType' object has no attribute 'info'
2019-12-14 15:43:03,926 [32mINFO[0m: => creating model 'resnet50'[0m
12/14/19 15:43:04.608 (P3981589.T3981589) [I] parrots exit with unhandled error
12/14/19 15:43:04.607 (P3981591.T3981591) [I] parrots exit with unhandled error
12/14/19 15:43:04.608 (P3981590.T3981590) [I] parrots exit with unhandled error
12/14/19 15:43:04.608 (P3981592.T3981592) [I] parrots exit with unhandled error
12/14/19 15:43:04.608 (P3981593.T3981593) [I] parrots exit with unhandled error
12/14/19 15:43:04.608 (P3981594.T3981594) [I] parrots exit with unhandled error
12/14/19 15:43:04.608 (P3981595.T3981595) [I] parrots exit with unhandled error
srun: error: SH-IDC1-10-5-31-36: task 7: Exited with exit code 1
srun: error: SH-IDC1-10-5-31-36: tasks 1-6: Exited with exit code 1
